packages/governance/src/types.js

```diff
@@ -688,7 +689,7 @@ export {};
  *
  * @param {ERef<ZoeService>} zoe
  * @param {import('@agoric/zoe/src/zoeService/utils.js').Instance<(zcf: ZCF<GovernanceTerms<{}>>) => {}>} allegedGoverned
- * @param {Instance} allegedGovernor
+ * @param {Instance<(zcf) => {publicFacet: GovernorPublic}>} allegedGovernor
```

Comment:
I'm pretty sure the allegedGovernor (here and in `AssertContractElectorate`) is an `instance`.  Doesn't this say it's a function?  The first use in both funct is
```
  const allegedGovernorPF = E(zoe).getPublicFacet(allegedGovernor);
```